pages/liquidate/index.tsx

```diff
@@ -353,14 +316,15 @@ const Liquidate: NextPage = () => {
           });
 
           if (transactionOutcome[0] == 'SUCCESS') {
+            if (walletPK) await fetchWalletBalance(walletPK);
             return toast.success('Bid placed, fetching chain data');
           } else {
-            return toast.error('Bid failed');
+            return toast.error('Bid failed', transactionOutcome);
           }
         } else if (type == 'increase_bid') {
           // if no user bid terminate action
           if (!userBid) return;
-
+          userBid = Number(userBid.toFixed(2));
```

Comment:
@bowtiedpyro sync with @yuriagile if this should be Number or BN

Reply:
will need to validate with Yuri, for now we can leave it like this